solidity/contracts/PCV.sol

```diff
@@ -10,6 +10,7 @@ import "./dependencies/CheckContract.sol";
 import "./dependencies/SendCollateral.sol";
 import "./interfaces/IPCV.sol";
 import "./token/IMUSD.sol";
+import "./interfaces/IMUSDSavingsRate.sol";
 
 contract PCV is CheckContract, IPCV, Ownable2StepUpgradeable, SendCollateral {
```

Comment:
Some of [the documentation](https://github.com/mezo-org/musd/blob/main/docs/README.md#protocol-controlled-value) will require an upgrade.